src/pull-request-data-provider.js

```diff
@@ -134,49 +131,49 @@ module.exports = class PullRequestDataProvider {
      */
 
     try {
-      const { repository } = await graphqlWithAuth(
-          `{
+      const {repository} = await graphqlWithAuth(
+        `{
             repository(owner: "${owner}", name: "${repo}") {
               pullRequest(number: ${pullRequestId}) {
                 bodyHTML
               }
             }
           }
-        `
+        `,
       );
```

Comment:
A good improve would be to extract these queries like a pageMap file and use them importing a JSON object, but well it looks not related to this PR too...

Reply:
As we have variables in the json object, I dont know how to do it by importing a JSON file

Reply:
easily doable with apollo but I don't know if you can do it with octokit